data/aliases.js

```diff
@@ -637,6 +639,7 @@ exports.BattleAliases = {
 	"se": "Stone Edge",
 	"spin": "Rapid Spin",
 	"sr": "Stealth Rock",
+	"st": "Spectral Thief",
```

Comment:
Context sensitive, but i suspect that when Marshadow stops to be newest thing, ST will stop meaning Spectral Thief.